ClientApp/KachnaOnline/src/app/home/events-overview/state-modal/state-modal.component.html

```diff
@@ -0,0 +1,22 @@
+<!--
+state-modal.component.ts
+Author: František Nečas
+-->
+<div class="modal-header">
+  <h4
+    class="modal-title">{{state.state == clubState.OpenBar ? "Otvíračka s baren" :
+    (state.state == clubState.OpenChillzone ? "Chillzóna" : "")}} {{state.start | date:'d. M.'}}</h4>
```

Comment:
The state could be a private event, too. Do we want to show these?

Reply:
Will private event be returned from the API anyway? @ondryaso 

If it is only returned for managers (which it should) then why not display it. Although it could be of a different color/the text message could be fixed... Or maybe it could be excluded

Reply:
I would show them, but with different colouring or any other way of indicating these states stand somewhat away from chill zone or open bar state types.

Reply:
If the request comes from an authenticated states manager, the response from `GET /states` includes private states. They should definitely be displayed.